src/index.ts

```diff
@@ -7,9 +7,7 @@ const app = createApp({
   trustProxy: true,
   // Default is 1MB
   // https://fastify.dev/docs/latest/Reference/Server/#bodylimit
-  // Here, we raise it to 50 MB (50 * 1024 * 1024).
-  // number is measured in Bytes
-  bodyLimit: 52428800,
+  bodyLimit: env.get().BODY_LIMIT,
```

Comment:
include this new variable in the docs

Reply:
This variable already exists in the docs, it just wasn't being used in this one spot